SU-SOE/soe_basic
# D8CORE-2249: Initial commit for the SOE Events theming.

# READY FOR REVIEW
- (Edit the above to reflect status)

# Summary
- Sub theme work for the SOE Events module.
- Changed the pages to account for the gradients in css behind the headers. Taking this approach since we will not have our row/cell variants ready when the sites launch.

# Needed By (Date)
- Tuesday

# Urgency
- High

# Steps to Test
1. Pull in this branch
2. Pull in SOE profile branch
3. Update configs
4. Recompile
5. Check on the Affect pages that they match the designs.

# Affected Projects or Products
- SOE Subtheming

# Affected Pages
/events/
/events/class (filtered lists)
/events/past
/event-series
/event/series/soe-event-series

# Associated Issues and/or People
- D8CORE-2247
- D8CORE-2248
- D8CORE-2249
- https://github.com/SU-SOE/soe_basic/pull/12
- Theming for the Events section
- @imonroe 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
